### ISSUE TITLE
Dissolve with multiples layers behaves wrongly

### ISSUE THREAD

**Issue by gamnamno** [jliljebl/flowblade#269]
Let's say you want to superimpose these 2 PNGs
![green](https://cloud.githubusercontent.com/assets/19158847/19620537/06fdd6c4-9880-11e6-9916-4c882f0fd107.png)
![red](https://cloud.githubusercontent.com/assets/19158847/19620539/0ec3b7fc-9880-11e6-9afe-0c6cf8a95164.png)
Here's a quick video that shows what happens :
https://youtu.be/scGMmBtD5w8

**Comment by jliljebl** [comment]
Just posting a video without a verbal clarification is basically just rude. Please add verbal description the issue in your view.

**Comment by jliljebl** [comment]
Is the problem in your view the halo like artifac. If so do this to get rid of it:
![screenshot from 2016-10-24 11-49-41](https://cloud.githubusercontent.com/assets/1823444/19639332/b939d6ba-99e0-11e6-9a18-f5f84b1a19dd.png)

Make both dissolves target same track. Halo comes from the first dissolve affecting the second image before dissolv. 

This is a deep MLT side issue, and cannot be changed in Flowblade source code. It is also quite certain that Dan will not be willing to change established compositing behavior for this Issue as that would potentially change every existing MLT application behavior.

**Comment by gamnamno** [comment]
Sorry, I thought it was clear enough on its own since superimposing images with alpha channels is a basic operation that we use a lot in special effects. You're right, your solution works, as this screenshot demonstrates a bit more clearly : 
![screenshot](https://cloud.githubusercontent.com/assets/19158847/19641992/7586b462-99e3-11e6-8850-d9836916a4b4.png)
I actually tried it previously and thought it was also not behaving as I wanted, but in fact it does. We just have to know that the layers order is reversed between the tracks and the image result.

Thanks for your help, I closed the issue.
